js/packages/openinference-instrumentation-llama-index/src/types.ts

```diff
@@ -17,3 +17,7 @@ export type RetrieverRetrieveMethodType = BaseRetriever["retrieve"];
 
 export type QueryEmbeddingMethodType =
   typeof llamaindex.BaseEmbedding.prototype.getQueryEmbedding;
+
+export type LLMChatMethodType = llamaindex.LLM["chat"];
+export type LLMCompleteMethodType = llamaindex.LLM["complete"];
+export type LLMObject = { metadata: llamaindex.LLMMetadata };
```

Comment:
this type seems a little bit odd - might just be a naming thing, will look at how it's used below but i would say LLMObject is a little to generic for this structure